tests/end-to-end/upgrade/internal/runner/runner.go

```diff
@@ -132,7 +132,11 @@ func (r *TestRunner) ExecuteTests(stopCh <-chan struct{}) error {
 }
 
 func (r *TestRunner) executeTaskFunc(taskHandler taskFn, stopCh <-chan struct{}, header, taskName string, createNs bool) bool {
-	taskLog := r.newLoggerForTask()
+	taskLog, err := r.newLoggerForTask()
+	if err != nil {
+		taskLog.Errorf("Cannot create uuid: %v", err)
```

Comment:
really🤔?

u cannot use the logger because u got error on creating it, so this obj will be nil.

so use `r.log`. Additionally, log more information e.g.
```go
r.log.Errorf("%s Cannot create uuid, the task won't be started, got err: %v", fullHeader, err)
```

Reply:
You're right. My mistake. Done!